app/jest.config.js

```diff
@@ -4,5 +4,6 @@ module.exports = {
   },
```

Comment:
Given you're using `jest-ts` without specifying a babel config, I think your tests don't go through Babel, so none of the Babel plugins/presets will be picked up in your tests. This might not be a problem for you, but if you're relying on `@emotion/babel-preset-css-prop` to work in your tests, you might hit an issue (e.g. `css` prop is not a valid attribute). This was a problem for us, so I thought I'd share